sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -958,6 +958,21 @@ case class Sort(
   override protected def withNewChildInternal(newChild: LogicalPlan): Sort = copy(child = newChild)
 }
 
+/**
+ * @param cluster The clustering expressions
+ * @param global  True means global clustering apply for entire data set,
+ *                False means clustering only apply within the partition.
```

Comment:
Do we really need this flag? Seems we always require within-partition clustering.

Reply:
Not sure if we'll need clustering globally in the future, remove this flag since now it's only used inside partition.